lighthouse-cli/test/fixtures/perf/preload_tester.js

```diff
@@ -8,6 +8,7 @@
 
 document.write('<script src="/perf/level-2.js?delay=500"></script>');
 document.write('<script src="/perf/level-2.js?warning&delay=500"></script>');
+document.write('<div id="animate-me"></div>')
```

Comment:
feels a bit odd to add it to the `preload.html` tester, I think `trace-elements.html` might be a more natural fit?

could just add an inline `<style>` over there with an animation